### PR TITLE
chore: Adding UTs and a bug fix

### PR DESCRIPTION
**Pull request by priteshbandi**
chore: Adding UTs and a bug fix.

Bug fixes: 
1. For `generate-signature` command the plugin framework was trying to parse parse input as `verify signature` command input. 
2. Removed end of line character for stdout response.

### CODE REVIEW

**Comment by priteshbandi** [comment]
Build is failing because of https://github.com/notaryproject/notation-plugin-framework-go/pull/11

**Review by rgnote — COMMENTED** [review]
Can you add details about the bug and the fix to the PR description?

**Review by rgnote — APPROVED** [review]
LGTM

File: cli/internal/mock/plugin.go [hunk 1]
```diff
@@ -23,24 +23,49 @@ import (
...
+	return &plugin.GenerateSignatureResponse{
+		KeyID:            "someKeyId",
+		Signature:        []byte("abcd"),
+		SigningAlgorithm: plugin.SignatureAlgorithmRSASSA_PSS_SHA256,
+		CertificateChain: [][]byte{[]byte("abcd"), []byte("wxyz")},
+	}, nil
 }
 
 func (p *mockPlugin) GenerateEnvelope(ctx context.Context, req *plugin.GenerateEnvelopeRequest) (*plugin.GenerateEnvelopeResponse, error) {
-	if p.fail {
```
Comment (Two-Hearts) [inline cli/internal/mock/plugin.go:42]: nit:
on line 72 `SignatureEnvelopeType: "application/jose+json",`, this should be
`SignatureEnvelopeType: req.SignatureEnvelopeType,`

File: cli/internal/mock/plugin.go [hunk 2]
```diff
@@ -50,8 +75,8 @@ func (p *mockPlugin) GenerateEnvelope(ctx context.Context, req *plugin.GenerateE
 }
 
 func (p *mockPlugin) VerifySignature(ctx context.Context, req *plugin.VerifySignatureRequest) (*plugin.VerifySignatureResponse, error) {
-	if p.fail {
-		return nil, fmt.Errorf("expected error")
+	if p.fail || !p.envGenerator {
```
Comment (Two-Hearts) [inline cli/internal/mock/plugin.go:78]: why do we need to check `!p.envGenerator` here?

**Review by JeyJeyGao — APPROVED** [review]
LGTM

File: cli/internal/mock/plugin.go (hunk 2 above)
Comment (priteshbandi) [reply cli/internal/mock/plugin.go]: Well we dont need this condition so removed.

**Pull request merged**